drivers/test_bus.py

```diff
@@ -1,22 +1,37 @@
 import unittest
 from unittest.mock import MagicMock
+from queue import Queue
 
-from drivers.bus import Bus
+from drivers.bus import BusHandler
 
 
-class BusTest(unittest.TestCase):
+class BusHandlerTest(unittest.TestCase):
 
     def test_publish(self):
         logger = MagicMock()
-        bus = Bus(logger)
+        bus = BusHandler(logger)
         bus.publish(1, b'some binary data')
         logger.write.assert_called_once_with(1, b'some binary data')
```

Comment:
I think here it should catch exception instead since no outputs are defined. In order for this not to fail it should contain this:

```python
bus = BusHandler(logger, out={1:[]})
```

That defines output channel named `1` that is not attached anywhere.